src/System.IO.Ports/ref/Configurations.props

```diff
@@ -6,4 +6,4 @@
       netfx;
     </BuildConfigurations>
   </PropertyGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
NIT: looks like this file didn't really change so perhaps we can undo the edit to this file.